Makefile

```diff
@@ -99,30 +99,46 @@ doc_R_pkg = \
 
 depends = .doc/$(1) .install/$(1) .check/$(1) .test/$(1)
 
+### Help
+
+help: ## Show this help message
+	@echo "Usage: make [target]"
+	@echo ""
+	@echo "Examples:"
+	@echo "  make all     "
+	@echo "  make document"
+	@echo ""
+	@echo "Notes:"
+	@echo "  - Components not included: cable (models), data.mining and DART (modules)."
+	@echo "  - Standard workflow: install packages, run checks, test, and document before submitting a PR."
+	@echo "  - Before submitting a PR, please ensure that all tests pass, code is linted, and documentation is up-to-date."
+	@echo ""
+	@echo "Available targets:"
+	@grep -h -E '^[a-zA-Z0-9_-]+:.*?## ' $(MAKEFILE_LIST) | \
+		awk 'BEGIN {FS = ":.*?## "}; {printf "  %-20s %s\n", $$1, $$2}'
 
 ### Rules
 
 .PHONY: all install check test document shiny \
             check_base check_models check_modules 
 
-all: install document
+all: install document ## Install all packages and generate documentation
 
-
-check_base: $(BASE_C) 
-check_models: $(MODELS_C) 
+check_base: $(BASE_C)     ## Check base packages  
+check_models: $(MODELS_C) ## Check model packages   
 
 # Install base first as Modules has a circular dependency on base,
 # and then run a check on modules
-check_modules: $(BASE_I) $(MODULES_C) 
+check_modules: $(BASE_I) $(MODULES_C) ## Install base packages, check module packages
+
+document: $(ALL_PKGS_D) .doc/base/all ## Generate documentation for all packages
+install: $(ALL_PKGS_I) .install/base/all ## Install all packages
+check: $(ALL_PKGS_C) .check/base/all     ## Check all packages
+test: $(ALL_PKGS_T) .test/base/all       ## Test all packages
+shiny: $(SHINY_I)                        ## Install Shiny app dependencies
 
-document: $(ALL_PKGS_D) .doc/base/all
-install: $(ALL_PKGS_I) .install/base/all
-check: $(ALL_PKGS_C) .check/base/all
-test: $(ALL_PKGS_T) .test/base/all
-shiny: $(SHINY_I)
 
-# Render the PEcAn bookdown documentation
-book:
+book: ## Render the PEcAn bookdown documentation
```

Comment:
This is a good specific example of my comment above about end-of-line comments making the code harder to read: When I open a new-to-me Makefile, one of the first things I skim for is short-named targets without dependencies, because they're often phony targets where the interesting work gets done. This change would make this entry harder for me to see while skimming.